Rocket-Nozzle-Design-and-Analysis/Rocket_Nozzle_Design_and_Analysis.tex

```diff
@@ -0,0 +1,230 @@
+}
+% page header for pages other than cover page
+\markboth{Project Design Document Standard}%
+{Parkus \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+  This project focuses on the design of a large-scale rocket nozzle capable of producing 100 Newtons of thrust. There will be a test stand used to measure the thrust force by method of a load cell.
+  There will be sensors placed in strategic locations that will assist in the data analysis to allow the team to understand conditions of the fluid as it passes through the nozzle and exits. The results
+  of the experiment will be summarized in a final report and presented at Imagine RIT, depending on the completion date.
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{
+    % If you include mathematical expressions or express variables in the design document, list them with their corresponding definitions here as a list.
+    % The two lines below make it look nice when defining units/values to constants.
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+\nomencalture{BoM}{Bill of Materials}
+% Below are examples of using nomenclature for math symbols and constants or units
+% \nomenclature{$\dot{m}$}{Mass flow rate
+%   \nomunit{\,\si{\kilo\gram\per\second}}}
+% \nomenclature{$c$}{Speed of light
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+\section{Introduction}\label{sec:introduction}
+\IEEEPARstart{T}{he} project will be concerned with the design, fabrication, and experimentation of a cold gas propulsion system. This is a propulsion system
+that is solely based off momentum transfer with little to no heat considerations. The project should have a team consisting of 10 to 15 members willing to dedicate
+5 to 10 hours a week to this project. The end result of the project will be a propulsion system, mounted on a test stand ready to test when the appropriate propellant
+is attached.
+
+\section{Primary Objective}\label{sec:primary-obj}
+The primary goal of this project is to use the analytical rocket equations to design a conical rocket nozzle to produce 100 Newtons of thrust.
+The team will design a new stand that is capable of securely fastening a propulsion test of 200 Newtons.
```

Comment:
I can say from experience that a test stand is its own bundle of ~~problems~~ joy. Since this PDD proposes them both, I strongly recommend having the test stand be a secondary objective and discuss its requirements and stuff on its own.

If you are using or modifying your existing test stand, cite that prior work.